admin_manual/configuration/files/external_storage/onedrive.rst

```diff
@@ -0,0 +1,74 @@
+==============================================
+How to Create and Configure Microsoft OneDrive
+==============================================
+
+To use Microsoft OneDrive as an external storage option in ownCloud, you need to
+do two things:
+
+#. `Create an application configuration`_
+#. `Configure a mount point in ownCloud`_
+
+Create an Application Configuration
+-----------------------------------
+
+To create an application configuration, open https://apps.dev.microsoft.com/ in your browser of choice and click "*Create App*" to begin creating the application.
+
+.. figure:: ../../../images/configuration/files/external_storage/onedrive/register-an-application.png
+   :alt: Register a new OneDrive application.
+
+The first thing to do is to set the application's name, located under "*Properties*". 
+After that, click "*Create*".
+
+After that, you can add the remaining application settings.
+Firstly, under "*Application Secrets*", click "*Generate New Password*", which generates a password for the application and displays, and displays it in a popup window.
```

Comment:
"and displays" is repeated

Reply:
oops. Fixing now.

Reply:
It doesn't read properly. Reworking it a bit further.